x-pack/plugins/fleet/server/services/epm/packages/get.ts

```diff
@@ -73,12 +74,30 @@ export async function getPackages(
   });
   // get the installed packages
   const packageSavedObjects = await getPackageSavedObjects(savedObjectsClient);
+  const MAX_PKGS_TO_LOAD_TITLE = 10;
 
-  const uploadedPackagesNotInRegistry = packageSavedObjects.saved_objects
-    .filter((pkg) => !registryItems.some((item) => item.name === pkg.id))
-    .map((pkg) =>
-      createInstallableFrom({ ...pkg.attributes, title: nameAsTitle(pkg.id), id: pkg.id }, pkg)
-    );
+  const packagesNotInRegistry = packageSavedObjects.saved_objects.filter(
+    (pkg) => !registryItems.some((item) => item.name === pkg.id)
+  );
+
+  const uploadedPackagesNotInRegistry = [];
+  for (const [i, pkg] of packagesNotInRegistry.entries()) {
```

Comment:
could this be done in parallel with `p-map` and a concurrency of 10?

Reply:
thanks for the tip, this is done